sdk/resourcemanager/package.json

```diff
@@ -32,7 +32,6 @@
     "yargs": "3.31.0",
     "npm-run-all": "4.1.5",
     "pom-parser": "1.2.0",
-    "request": "2.88.2",
     "request-promise": "4.2.6",
```

Comment:
Do we need `request-promise`?

Reply:
Emmm, seems it's also deprecated:
https://www.npmjs.com/package/request-promise

Let me replace it with axios.

Reply:
If we use it, we can keep it.

However, we may want to find an alternative. As request is deprecated, this lib (a wrapper) is also deprecated...
https://www.npmjs.com/package/request-promise?activeTab=readme

Reply:
switched to axios